### ISSUE TITLE
[A11Y] Combobox - State of active component lacks 3 to 1 contrast ratio

### ISSUE THREAD

**Issue by chornonoh-vova** [dequelabs/cauldron#1637]
## Issue Details

State of active component lacks 3 to 1 contrast ratio

---

### Short Description

The state of an ACTIVE user interface component does not have at least a 3 to 1 luminosity contrast ratio with either the inner or outer adjacent background.
Element description: Tick icon color on hover.

Element location: "Scan or scan group" and "Role" drop downs -> Expand and select an option -> Tick Icon appears for the selected option.

Icon color on hover: #57A711
Background color: #F2F2F2
The contrast ratio is: 2.7:1

### Impact

- [x] Blocker
- [ ] Critical
- [ ] Serious
- [ ] Moderate
- [ ] Minor

### Current Behavior:

Color of the icon doesn't have enough contrast with the background

### Expected Behavior:

Color of the icon has enough contrast with the background

---

### Auditor ID / URL

https://axeauditor.dequecloud.com/test-run/2783be16-2a07-11ee-91e9-c3d20c45179b/issue/a21189ec-4fc2-11ef-a633-f3bac32783d0

### Additional Details

<details>
<summary>Expand for extra context</summary>

- [x] Discovered during VPAT
- [ ] Found using NVDA
- [ ] Found using Chrome Screen Reader
- [ ] Found with special configuration 
</details>

---

## Screenshots 

![image](https://github.com/user-attachments/assets/d3bf84a2-dfde-49f0-a15d-5e4f45f94f4c)

---

### 1. Perceivable

<details> 
<summary>Expand</summary>

#### 1.1 Text Alternatives

- [ ] 1.1.1 Non-text Content

#### 1.2 Time-based Media

- [ ] 1.2.1 Audio-only and Video-only (Prerecorded)
- [ ] 1.2.2 Captions (Prerecorded)
- [ ] 1.2.3 Audio Description or Media Alternative (Prerecorded)
- [ ] 1.2.4 Captions (Live)
- [ ] 1.2.5 Audio Description (Prerecorded)

#### 1.3 Info and Relationships

- [ ] 1.3.1 Info and Relationships
- [ ] 1.3.2 Meaningful Sequence
- [ ] 1.3.3 Sensory Characteristics
- [ ] 1.3.4 Orientation
- [ ] 1.3.5 Identify Input Purpose

#### 1.4 Distinguishable

- [ ] 1.4.1 Use of Color
- [ ] 1.4.2 Audio Control
- [ ] 1.4.3 Contrast (Minimum)
- [ ] 1.4.4 Resize text
- [ ] 1.4.5 Images of Text
- [ ] 1.4.10 Reflow
- [x] 1.4.11 Non-text Contrast
- [ ] 1.4.12 Text Spacing
- [ ] 1.4.13 Content on Hover or Focus
</details>

### 2. Operable

<details> 
<summary>Expand</summary>

#### 2.1 Keyboard Accessible

- [ ] 2.1.1 Keyboard
- [ ] 2.1.2 No Keyboard Trap
- [ ] 2.1.4 Character Key Shortcuts

#### 2.2 Enough Time

- [ ] 2.2.1 Timing Adjustable
- [ ] 2.2.2 Pause, Stop, Hide

#### 2.3 Seizures and Physical Reactions

- [ ] 2.3.1 Three Flashes or Below Threshold

#### 2.4 Navigable

- [ ] 2.4.1 Bypass Blocks
- [ ] 2.4.2 Page Titled
- [ ] 2.4.3 Focus Order
- [ ] 2.4.4 Link Purpose (In Context)
- [ ] 2.4.5 Multiple Ways
- [ ] 2.4.6 Headings and Labels
- [ ] 2.4.7 Focus Visible

#### 2.5 Input Modalities

- [ ] 2.5.1 Pointer Gestures
- [ ] 2.5.2 Pointer Cancellation
- [ ] 2.5.3 Label in Name
- [ ] 2.5.4 Motion Actuation
</details>

### 3. Understandable

<details> 
<summary>Expand</summary>

#### 3.1 Readable

- [ ] 3.1.1 Language of Page
- [ ] 3.1.2 Language of Parts

#### 3.2 Predictable

- [ ] 3.2.1 On Focus
- [ ] 3.2.2 On Input
- [ ] 3.2.3 Consistent Navigation
- [ ] 3.2.4 Consistent Identification

#### 3.3 Input Assistance

- [ ] 3.3.1 Error Identification
- [ ] 3.3.2 Labels or Instructions
- [ ] 3.3.3 Error Suggestion
- [ ] 3.3.4 Error Prevention (Legal, Financial, Data)
</details>

### 4. Robust

<details>
<summary>Expand</summary>

#### 4.1 Compatible

- [ ] 4.1.1 Parsing
- [ ] 4.1.2 Name, Role, Value
- [ ] 4.1.3 Status Messages
</details>

**Comment by scurker** [comment]
The icon color needs to change to:

- Light Mode: `accent-primary`
- Dark Mode: `accent-info`
